cli/popper/commands/cmd_run.py

```diff
@@ -153,13 +165,21 @@ def run_pipeline(action, wfile, skip, workspace, reuse,
         log.warn("Using --parallel may result in interleaved output. "
                  "You may use --quiet flag to avoid confusion.")
 
+    if skip_clone:
+        log.warn("Please make sure that all the required action repositories "
+                 "are available locally.")
+
+    if skip_pull:
+        log.warn("Please make sure that all the required images are "
+                 "present locally.")
```

Comment:
these might get too repetitive for someone running with `--skip-pull` or `--skip-clone` over and over again. I'd vote for leaving these warning messages out; you have implemented proper messages that are shown when an image doesn't exist and `--skip-pull` is used, which is informative enough